### DISCUSSION TITLE
SimpleAdmin errors?  development branch

### DISCUSSION THREAD

**Discussion by buggz2k** [iamromulan/cellular-modem-wiki#2] [General - Quectel]
Hello,

Anyone else get errors in Simple Admin development branch in the following menu items?
 - Simple Network tab - Scan Neighbour Cells -
I get, Request Error

 - Simple Network tab - Scan NSA Cells
I get, Request Error

 - Simple Network tab - Full Network Scan
I get, Request Error
this probably due to using the same commands as the above two are using?

**Comment by iamromulan** [comment]
Development branch and main branch are pretty much in sync right now.

There's some new stuff I've been working on in there (simpleupdates) but it's not part of the toolkit script yet.
@dr-dolomite added in the new scan features however I don't think this is going to be the final version. He's working on a new simpleadmin 2.0 that handles parsing client side. 
Which is why I started to focus on creating an automatic update system. I personally haven't used the scan features yet but you can achieve this through the AT+QSCAN command in the at command section.

This project is moving forward and is always improving so stay tuned. 😁

**Comment by dr-dolomite** [comment]
> Hello,
> 
> Anyone else get errors in Simple Admin development branch in the following menu items?
> 
> * Simple Network tab - Scan Neighbour Cells -
>   I get, Request Error
> * Simple Network tab - Scan NSA Cells
>   I get, Request Error
> * Simple Network tab - Full Network Scan
>   I get, Request Error
>   this probably due to using the same commands as the above two are using?

What module are you using? Since AT commands may change from one version to another, it is possible that the commands may not work properly. (Example, QNWLOCK changing "common/lte" to "common/4g".

**Comment by iamromulan** [comment]
@dr-dolomite I may have to add logic to the toolkit for a different set of files to be downloaded dependent on the module being installed to. I know for sure this will be a must when the quad core 64bit processors appear in the RM5 series.

So far everything has been pretty cross compatible.

**Reply by dr-dolomite** [reply]
Yes we can do that later on. Also, please try the commands manually OP on AT commander if these commands are actually supported on your module.

AT+QSCAN=3,0
AT+QENG="neighbourcell"

**Reply by buggz2k** [reply]
Okay, I verfied that AT command:
AT+QENG="neighbourcell"

Does NOT return any result.
All of the following commands work:
ATI

Quectel
RM520N-GL
Revision: RM520NGLAAR01A07M4G

OK

AT+QSCAN=3,0

+QSCAN: "NR5G",310,260,389070,183,-100,-12,20,0
+QSCAN: "NR5G",310,260,521310,772,-113,-14,-2,1
+QSCAN: "NR5G",310,260,501390,772,-114,-15,-2,1
+QSCAN: "NR5G",310,260,126510,168,-82,-11,35,0
+QSCAN: "NR5G",313,340,129370,328,-103,-15,18,0
+QSCAN: "NR5G",310,410,397470,423,-103,-15,-8,0
+QSCAN: "NR5G",313,340,123870,426,-125,-16,-5,0
+QSCAN: "LTE",310,260,5035,138,-87,-10,37,23
+QSCAN: "LTE",310,410,5110,87,-100,-16,30,13
+QSCAN: "LTE",313,100,5110,87,-100,-16,30,13
+QSCAN: "LTE",310,260,66786,102,-97,-17,28,18
+QSCAN: "LTE",310,260,66661,102,-98,-6,-8,0
+QSCAN: "LTE",311,480,5230,148,-84,-10,44,118
+QSCAN: "LTE",311,480,8540,148,-84,-10,-4,0

OK


at+qeng"seringcell"

+QENG: "servingcell","NOCONN","NR5G-SA","FDD",310,260,186DA6017,183,772700,389070,25,3,-101,-12,15,0,-

OK

AT+QNCF="nr5g_cell_id"

+QNWCFG: "nr5g_cell_id",130062186DA6017,186DA6017,186DA6

OK

**Reply by dr-dolomite** [reply]
I see, thanks for this. Apparently, this is pretty normal. When the modem uses NR5G-SA despite having AUTO on your network preferred mode, it will not scan nearby LTE bands through neighbourcell command. This will only work with LTE and NSA network modes. No worries there.

**Comment by buggz2k** [comment]
Thanks for the replies.
I have been offline for a while now, hope to answer the questions posted to me soon.

I am very happy to see this package is still being developed!
Is there a donation page to keep this project going?
I would gladly donate to this project.

Thanks!

**Comment by iamromulan** [comment]
> Thanks for the replies.
> I have been offline for a while now, hope to answer the questions posted to me soon.
> 
> I am very happy to see this package is still being developed!
> Is there a donation page to keep this project going?
> I would gladly donate to this project.
> 
> Thanks!
> 
> 

We don't have a main donation page *yet* not sure if we will move to do that or not, however,
My Venmo: "@iamromulan"
@dr-dolomite (Who is doing some major work on the UI) has a Ko-Fi https://ko-fi.com/drdolomite
